eng/pipelines/compliance.yml

```diff
@@ -27,11 +27,42 @@ jobs:
           targetArgument: "$(Build.SourcesDirectory)"
           result: "PoliCheck.xml"
 
+      - task: DownloadPipelineArtifact@2
+        displayName: "Download symbols - RTM artifact"
+        inputs:
+          source: specific
+          project: 'devdiv'
+          pipeline: NuGet.Client-Official
+          artifact: "symbols - RTM"
+          path: $(Pipeline.Workspace)\symbols\RTM
+
+      - task: DownloadPipelineArtifact@2
+        displayName: "Download symbols - NonRTM artifact"
+        inputs:
+          source: specific
+          project: 'devdiv'
+          pipeline: NuGet.Client-Official
+          artifact: "symbols - NonRTM"
```

Comment:
Perhaps it's worthwhile putting comments in the Build_And_UnitTest.yml file where the RTM & NonRTM build artifacts are published, saying that `compliance.yml` downloads the artifacts, so if the artifact name is changed, it needs to be changed in both places.  Otherwise we won't know until the change is merged and the pipeline starts failing.

Same with the RTM symbols

Reply:
Ok. Removed `RTM` artifact download.